src/components/Nav/Masthead/HelpDropdown.tsx

```diff
@@ -6,12 +6,12 @@ import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
 import { QuestionCircleIcon } from '@patternfly/react-icons/';
 import { connect } from 'react-redux';
 import { isUpstream } from '../../UpstreamDetector/UpstreamDetector';
-import { Status, Component, StatusKey } from '../../../types/StatusState';
+import { Status, ExternalServiceInfo, StatusKey } from '../../../types/StatusState';
 import { config } from '../../../config';
 
 type HelpDropdownProps = {
   status: Status;
-  components: Component[];
+  components: ExternalServiceInfo[];
```

Comment:
Minor, but just for consistency:
![image](https://user-images.githubusercontent.com/1662329/132197191-1787490e-3732-4cce-b1f6-c71f67f26620.png)

if the renaming Component -> ExternalServiceInfo is to add clarity (+1 on that), I'd also propagate to the properties that is used if it's a direct reference as it shows here.

(Non a blocker from my side, it can optional to do it in the future, code is clear to me).